packages/realm/src/Realm.ts

```diff
@@ -1316,582 +1141,135 @@ function isEmbedded(objectSchema: binding.ObjectSchema): boolean {
+  export import PropertiesTypes = internal.PropertiesTypes;
+  export import PropertySchema = internal.PropertySchema;
+  export import PropertySchemaParseError = internal.PropertySchemaParseError;
+  export import PropertySchemaShorthand = internal.PropertySchemaShorthand;
+  export import PropertySchemaStrict = internal.PropertySchemaStrict;
+  export import PropertyTypeName = internal.PropertyTypeName;
+  export import ProviderType = internal.ProviderType;
+  export import ProxyType = internal.ProxyType;
+  export import RealmEventName = internal.RealmEventName;
+  export import RealmObjectConstructor = internal.RealmObjectConstructor;
+  export import RelationshipPropertyTypeName = internal.RelationshipPropertyTypeName;
+  export import Results = internal.Results;
+  export import SchemaParseError = internal.SchemaParseError;
+  export import SessionState = internal.SessionState;
+  export import SessionStopPolicy = internal.SessionStopPolicy;
+  export import Set = internal.RealmSet;
+  export import SortDescriptor = internal.SortDescriptor;
+  export import SSLConfiguration = internal.SSLConfiguration;
+  export import SSLVerifyCallback = internal.SSLVerifyCallback;
+  export import SSLVerifyObject = internal.SSLVerifyObject;
+  export import SubscriptionSetState = internal.SubscriptionSetState;
+  export import SyncConfiguration = internal.SyncConfiguration;
+  export import SyncError = internal.SyncError;
+  export import UpdateMode = internal.UpdateMode;
+  export import UserChangeCallback = internal.UserChangeCallback;
+  export import UserState = internal.UserState;
+  export import User = internal.User;
+  export import WaitForSync = internal.WaitForSync;
+  export import GeoBox = internal.GeoBox;
+  export import GeoCircle = internal.GeoCircle;
+  export import GeoPoint = internal.GeoPoint;
+  export import GeoPolygon = internal.GeoPolygon;
+  export import CanonicalGeoPolygon = internal.CanonicalGeoPolygon;
+  export import CanonicalGeoPoint = internal.CanonicalGeoPoint;
+  export import GeoPosition = internal.GeoPosition;
+
+  // Deprecated exports below
+  /** @deprecated Will be removed in v13.0.0. Please use {@link internal.AppServicesFunction} */
+  export import RealmFunction = internal.AppServicesFunction;
+  /** @deprecated Will be removed in v13.0.0. Please use {@link internal.CanonicalPropertySchema} */
+  export import CanonicalObjectSchemaProperty = internal.CanonicalPropertySchema;
+  /** @deprecated Will be removed in v13.0.0. Please use {@link internal.ClientResetRecoverUnsyncedChangesConfiguration} */
+  export import ClientResetRecoveryConfiguration = internal.ClientResetRecoverUnsyncedChangesConfiguration;
+  /** @deprecated Will be removed in v13.0.0. Please use {@link internal.PropertySchema} */
+  export import ObjectSchemaProperty = internal.PropertySchema;
+  /** @deprecated Will be removed in v13.0.0. Please use {@link internal.RealmObjectConstructor} */
```

Comment:
I'm not sure I understand the context for this suggestion.

Reply:
When hovering it will include `internal`, but with the suggested changes it will only show the type without the `internal` part, but still correctly linking it.